infrastructure/critical/modules/rds/security_groups.tf

```diff
@@ -0,0 +1,30 @@
+resource "aws_security_group" "database_sg" {
+  description = "controls direct access to application instances"
```

Comment:
Does it actually, or have you copy/pasted this without checking it?

Reply:
PS Security descriptions are a massive faff to change after the fact, so let's get it right now!

Reply:
> Does it actually, or have you copy/pasted this without checking it?

Correcto

Reply:
Updated, does that make more sense?